### PR TITLE
feature: implement adding http header and adding multiple http header…

### PR DESCRIPTION
**Pull request by helintongh**
…s options

### CODE REVIEW

File: press_tool/util.h [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
+    rt = (double(bytes) / KB_BYTE);
+  }
+  else {
+    suffix = "bytes";
+    rt = (double)bytes;
+  }
+  return (std::to_string(rt) + suffix);
+}
+
+std::vector<std::string> &split(std::string &str, const std::string &delimiter, std::vector<std::string> &elems) {
```
Comment (qicosmos) [inline press_tool/util.h:32]: inline

Comment (qicosmos) [reply press_tool/util.h:32]: do you deal with trim?

Comment (helintongh) [reply press_tool/util.h]: fix it,already deal trim.

**Review by qicosmos — APPROVED** [review]
LGTM

**Pull request merged**